README.md

```diff
@@ -1,109 +1,87 @@
-# carbon-multitenancy
+# Carbon Multitenancy
 
 ---
-
-|  Branch | Build Status |
+|  Branch       | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/carbon-multitenancy/badge/icon)](https://wso2.org/jenkins/job/carbon-multitenancy) |
-
-
+| master        | [![Build Status](https://wso2.org/jenkins/job/carbon-multitenancy/badge/icon)](https://wso2.org/jenkins/job/carbon-multitenancy) |
 ---
 
 Latest Released Version v4.5.0, v4.4.4.
 
-#### carbon-multitenancy repo contains the the following component.
+Carbon servers have become container native with the introduction of Carbon v5 (C5). As a result, In-JVM multi-tenancy 
+model which was used in previous Carbon versions was removed and container multitenancy was introduced. In C5,
```

Comment:
Thanks for the feedback Irunika! In this sentence the subject is "In-JVM multi-tenancy model" that why it was written as singular.